external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -90,10 +90,13 @@ use move_command_line_common::files::FileHash;
 use move_compiler::{
```

Comment:
These changes all seem coherent, though I wonder if there is an opportunity to write a helper or two to simplify some of the code.

Reply:
I was thinking the same thing, but honestly I could not find a sensible way to helper-ize it as most additional are not easily separable from their enclosing context. Happy to follow suggestions, though!